sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/append_blob_client.hpp

```diff
@@ -119,8 +119,8 @@ namespace Azure { namespace Storage { namespace Blobs {
      * exists.
      *
      * @param options Optional parameters to execute this function.
-     * @return A CreateAppendBlobResult describing the newly created append blob. Null if the blob
-     * already exists.
+     * @return A CreateAppendBlobResult describing the newly created append blob.
+     * CreateAppendBlobResult.Created is false if the blob already exists.
```

Comment:
This looks like a behavioral change, worth adding a change log entry, maybe under "other changes"?

Reply:
Yes, it is a behavioral change. But we just added this API in the same version, without shipping a release.